website/docs/reference/transforms/rename_fields.md.erb

```diff
@@ -32,7 +32,18 @@
 
 ### Key Conflicts
 
-Keys specified in this transform will replace existing keys.
+Keys specified in this transform will replace existing keys _in full_. For
+example, if a target key contains an object value it will be replaced _entirely_
+by the new key's value, even if the new value is an object itself. Vector will
+_not_ perform a deep merge.
+
+<Alert type="warning">
+
+Vector makes no guarantee on the order of execution. If two rename operations
```

Comment:
Is this for some future feature? Because we currently do, we rename in the order given.

Reply:
No, but we aren't representing the order in TOML. Are table attributes deserialized in an ordered data structure? Generally, this is not the case and to my knowledge, this is not part of the TOML spec.

Reply:
Correct, https://github.com/toml-lang/toml/blob/master/versions/en/toml-v0.5.0.md#user-content-table doesn't discuss ordering.